pkg/cloudprovider/types.go

```diff
@@ -78,6 +78,20 @@ func (its InstanceTypes) OrderByPrice(reqs scheduling.Requirements) InstanceType
 	return its
 }
 
+// FilterByCapacityType filters the list of instanceTypes based on the provided capacityType and supported zones in the requirements
+func (its InstanceTypes) FilterByCapacityType(requirements scheduling.Requirements, capacityType string) InstanceTypes {
```

Comment:
Can we hoist `type InstanceType struct` above these helpers. This file gets a lot of eyes, so I'd like to make the data definition as clear as possible.

Reply:
Got it. Addressed.